src/Avalonia.Base/Data/BindingPriority.cs

```diff
@@ -35,6 +35,11 @@ public enum BindingPriority
         /// A style binding.
         /// </summary>
         Style,
+        
+        /// <summary>
+        /// The value is inherited from an ancestor element.
+        /// </summary>
+        Inherited,
```

Comment:
Isn't it internal only state? As I see so far, developer can't get priority of the property value from the public API, and calling SetValue with this makes less sense.

Reply:
Not really, because it is exposed on the property change event args, where it's a valid value. Also we already have `Unset` priority in this enum. We should probably add a check to ensure that no-one tries to set/bind properties with these enum values though.

Reply:
Done.